lrucache_test.go

```diff
@@ -28,8 +29,91 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	dto "github.com/prometheus/client_model/go"
 )
 
+func TestLRUCacheUnexpiredEvictionMetrics(t *testing.T) {
```

Comment:
This test seems to have two test cases in one that should be separated:

1. Adding an item to full cache after some items have expired = no evictions.
2. Adding an item to full cache with no item expirations = 1 eviction.

Reply:
Updated to move this into two different t.Run blocks in TestLRUCache()